src/styles/sidebar/components/markdown-editor.scss

```diff
@@ -1,3 +1,4 @@
+@use "../../mixins/buttons";
```

Comment:
This fixes an unrelated issue. When you click the tool buttons with the mouse, the blue focus ring shows up. This fixes that. I think it might be the last place in our code where we have that?  A further improvement could be to revisit this when we convert more things to use `<Button>`

Reply:
Some of the specific styles (eg. `border: none`) may no longer be needed with this change.

Reply:
good catrch